src/Service/UpdateVideoItem/UpdateVideoItemRequest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\media_mpx\Service\UpdateVideoItem;
+
+use Drupal\media\Entity\Media;
+use GuzzleHttp\Psr7\Uri;
+
+/**
+ * Contains the data needed to update a media entity via UpdateVideoItem.
+ *
+ * @package Drupal\media_mpx\Service\UpdateVideoItem
+ */
+class UpdateVideoItemRequest {
+
+  const MPX_MEDIA_DATA_SERVICE_URL = "http://data.media.theplatform.com/media/data/Media/";
```

Comment:
This is not actually a constant, and there's an API for this as the hostname may change. However the given id for an mpx object will be static once it's set.

Actually, I'm not sure this is needed. You're concatenating this to the full id below, which I think will duplicate this portion.

Reply:
It won't duplicate it, as I'm using ->load() which expects the Uri already.... Buuut I just noticed there's a `loadByNumericId` method, which indeed makes this oddity here totally unnecessary. Fixing 👍